nisync/session.py

```diff
@@ -864,6 +864,10 @@ def time_references(self):
     def resource_name(self):
         return self._resource_name
 
+    @property
+    def session_handle(self):
+        return _visatype.ViSession(self._vi)
```

Comment:
We should just return self._vi.
Let clients worry about converting it to a ctype, if they need to.

nimi-python should automatically generate the code for conversion to ctypes in the LibararyInterpreter. For grpc, if we support it in the future, we don't pass ctypes.

Reply:
I removed the parsing in the function.